include/oboe/AudioStream.h

```diff
@@ -376,6 +376,17 @@ class AudioStream : public AudioStreamBase {
 
 protected:
 
+    /**
+     * This is used to detect more than one error callback from a stream.
+     * These were bugs in some versions of Android that caused multiple error callbacks.
+     * Internal bug b/63087953
+     *
+     * @return true if the error callback was already called, otherwise false
+     */
+    bool wasErrorCallbackCalled() {
```

Comment:
Document that this method is also what is responsible for setting the atomic bool

Reply:
Done.